frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/ticketGroups/groups/addOrEditGroup/groupSettingsForm.component.tsx/groupSettingsForm.styles.ts

```diff
@@ -0,0 +1,91 @@
+/**
+ *  Copyright (C) 2023 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { FONT_WEIGHT } from '@/v5/ui/themes/theme';
+import { Toggle as BaseToggle } from '@controls/inputs/toggle/toggle.component';
+import { Box, InputLabel } from '@mui/material';
+import styled from 'styled-components';
+
+export const FormBox = styled(Box)`
```

Comment:
Is there a particular reason why you used MUI `Box` instead of a `div`?

Reply:
I was playing around with adding this styling to the theme so that it could be reused but never did. I'll turn it back into a div